tests/cxx/unit_tests/ghostfragment/capping/DCLC.hpp

```diff
@@ -0,0 +1,121 @@
+#pragma once
+
+#include <catch2/catch.hpp>
+#include <chemist/molecule/atom.hpp>
+#include <chemist/molecule/molecule.hpp>
+#include <chemist/nucleus/nuclei.hpp>
+#include <src/ghostfragment/capping/DCLC.cpp>
+
+namespace testing {
+// Correct answers for three hydrocarbons -- ethane with two
+// fragments, propane with two fragments, and propane with four
+// fragments. Answers generated in Mathematica. 
+
+// Ethane CapSets (2 carbon, 2 frags)
+std::vector<chemist::CapSet> Ethane2() {
```

Comment:
The function names in this file are IMHO confusing. I'd recommend something like `ethane_dclc_caps`.